masterbase/steam.py

```diff
@@ -219,10 +221,13 @@ def add_nand_filter(self) -> None:
         raise NotImplementedError
 
 
-def get_ip_as_integer(ip: str) -> str:
-    """Get the fake IP for a server. Wack math from ChatGPT and @Saghetti."""
-    ip_parts = [int(part) for part in ip.split(".")]
-    return (ip_parts[0] * 256**3) + (ip_parts[1] * 256**2) + (ip_parts[2] * 256) + ip_parts[3]
+def get_ip_as_integer(ip: str) -> int:
+    """Get the fake IP for a server as a word.
+
+    Wack math from ChatGPT and @Saghetti.
+    """
+    ip_parts = np.array([int(part) for part in ip.split(".")])
+    return int(np.bitwise_or.reduce(ip_parts << np.arange(24, -1, -8)))
```

Comment:
much cleaner

Reply:
also, while we are here, would be epic to add a unit test for this function 🤠

Reply:
see https://github.com/MegaAntiCheat/masterbase/pull/58/commits/df7f4d4d29ec6363961b96683eb62ba1c81bda76